src/settings.ts

```diff
@@ -25,13 +25,13 @@
  */
 
 module powerbi.extensibility.visual {
+    // powerbi.extensibility.utils.dataview
     import DataViewObjectsParser = powerbi.extensibility.utils.dataview.DataViewObjectsParser;
 
     export class GlobeMapSettings extends DataViewObjectsParser {
         public dataPoint: DataPointSettings = new DataPointSettings();
     }
 
     export class DataPointSettings {
```

Comment:
Yes, but we will can to add new settings later and it's easier with separate settings file